staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -618,7 +665,7 @@ func TestCacherNoLeakWithMultipleWatchers(t *testing.T) {
 				return
 			default:
 				ctx, _ := context.WithTimeout(context.Background(), 3*time.Second)
-				w, err := cacher.Watch(ctx, "pods/ns", "0", pred)
+				w, err := cacher.Watch(ctx, "pods/ns", storage.ListOptions{ResourceVersion: "0", Predicate: storage.Everything})
```

Comment:
Please use pred - this test was added to catch a leak in bookmarks, which are set in `pred`, but not now (in storage.Everything).

Reply:
Good catch. Fixed.